src/storage/db_interface_compare.py

```diff
@@ -96,3 +95,13 @@ def get_ssdeep_hash(self, uid):
     def get_entropy(self, uid):
         file_object_entry = self.file_objects.find_one({'_id': uid}, {'processed_analysis.unpacker.entropy': 1})
         return file_object_entry['processed_analysis']['unpacker']['entropy'] if 'unpacker' in file_object_entry['processed_analysis'] else 0.0
+
+    def get_exclusive_files(self, compare_id: str, root_uid: str) -> List[str]:
```

Comment:
I am not so happy with this decision since this a functionality that is only used in the compare plug-in. We should discuss this next week.

Reply:
The problem is: In its previous location it's almost impossible to test